configs/prod/roles/notifications.json

```diff
@@ -19,7 +19,7 @@
       "name": "Notifications viewer",
       "description": "Read only access to notifications and integrations applications.",
       "system": true,
-      "platform_default": true,
+      "platform_default": false,
       "version": 1,
```

Comment:
Sure, you might also need to update the version of the roles in order to update the role.

BTW, current seeding process is very slow, so we paused any new seedings. We are in the process of migrating to serve from public schema, which will solve this problem. Hopefully which will be done before next week. @coderbydesign @wcmitchell

Reply:
IIUC, we might need to bump the version to `2` in both files.

@aferd  could you update?

> BTW, current seeding process is very slow, so we paused any new seedings.

Understood, thanks!